app/views/shared/banners/_global_notice_banner.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <div class="govuk-notification-banner" role="region" aria-labelledby="govuk-notification-banner-title" data-module="govuk-notification-banner">
```

Comment:
Can we use the [notification banner component](https://github.com/DFE-Digital/govuk-components/blob/555aec0b23958bc6530e29491a11c08852753403/app/components/govuk_component/notification_banner.rb) for this?